packages/jellyfish-api-core/__tests__/category/loan/paybackLoan.test.ts

```diff
@@ -752,14 +836,354 @@ describe('paybackLoan failed', () => {
     await expect(promise).rejects.toThrow(RpcApiError)
     await expect(promise).rejects.toThrow('tx must have at least one input from token owner')
   })
+})
 
-  it('should fail paybackLoan if resulted in zero interest loan', async () => {
-    const promise = bob.rpc.loan.paybackLoan({
-      vaultId: bobVaultId,
-      amounts: '40@TSLA',
-      from: bobloanAddr
+describe('paybackLoan before FortCanningHeight', () => {
+  let tGroupFCH: TestingGroup
+  let loanTokenMinter: Testing
+  let testing: Testing
+  let colAddr: string
+  let vaultId: string
+  const tslaLoanAmount = 10
+
+  async function setupPreFCHContainer (fchBlockHeight: number): Promise<void> {
+    tGroupFCH = TestingGroup.create(2)
+    const hardForks: HardForkOptions[] = [{ name: 'fortcanninghillheight', blockHeight: fchBlockHeight }]
+    await tGroupFCH.start({ hardForks })
+    loanTokenMinter = tGroupFCH.get(0)
+    testing = tGroupFCH.get(1)
+    await tGroupFCH.start()
+    await loanTokenMinter.container.waitForWalletCoinbaseMaturity()
+    await tGroupFCH.waitForSync()
+  }
+
+  async function setupPostFCHContainer (): Promise<void> {
```

Comment:
redundant. `setupPreFCHContainer` can be used to do the same task by passing the height